tests/unit/disco/containers/TestDiscoPane.js

```diff
@@ -241,7 +247,7 @@ describe(__filename, () => {
           guid: '@guid1',
           slug: 'discovery-addon-1',
         },
-      }]));
+      }]), { params: { platform: 'Darwin' } });
```

Comment:
mmh this looks weird here. What is tested?

Reply:
Oops it's wrong and should have been removed but was before I was just adding `ownProps` everywhere. I'll remove it 😄